README.md

```diff
@@ -23,7 +26,9 @@ Whack something like this in your `accessories: []` section of homebridge config
     "manufacturer": "Fujitsu APC",
     "model": "FJT750i",
     "serial": "AS1305696928",
-    "interval": 1
+    "interval": 1,
+    "logging": false,
+    "temperatureSensor": false
```

Comment:
Is it possible just to detect the sensor and add it if the users UPS supports it?

Reply:
Hm. Good question. Without one fancy enough to test, I'm happy to leave this opt in for now. There are likely many more features like this too!

Reply:
I'm wondering if mine does now. I will take a look when you publish this.

I just implanted device feature sniffing in another home bridge plugin:
https://github.com/davidjbradshaw/homebridge-aeg-wellbeing/blob/master/src/index.ts#L27

Reply:
Oh that's clever - take one API sample, see what it returns, and proceed from there. I like it!